packages/entities/entities-shared/src/components/entity-empty-state/EntityEmptyState.vue

```diff
@@ -12,7 +12,7 @@
         v-if="title || $slots.title"
         class="entity-empty-state-title"
       >
-        <h1>
+        <h1 :class="appearance">
```

Comment:
issue: you need to map the class to a computed property; otherwise, if an invalid appearance is passed it will be bound as a class and styled incorrectly.